### DISCUSSION TITLE
part-ii/2018-0/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#95] [Question comments]
Part II 2018 Algebraic Geometry: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2018-0/

**Comment by Kiolt** [comment]
<details>
<summary>Last part.</summary>

I found the degree to be 3 by looking at the preimage of $(0:1)$. We can find ramification points in the affine patch by looking at the discriminant and ramification degrees by asking when do we have a triple root. I dealt with the points at infinity by hand to confirm the degree. Another approach might be to find 3 points which aren't branch points and show that their preimage in the affine patch consists of 3 points, then use a parity argument in Riemann-Hurwitz for the points at infinity.

</details>

**Reply by eha1234** [reply]
<details><summary>[spoiler title]</summary>

I got the genus to be 3 using this?
I used that there are only 2 points at infinity, so it must be that one has ramification index 2, the other has ramification index 1. 
</details>
